test/runtests.jl

```diff
@@ -43,6 +43,7 @@ a1 = Graph(adjmx1)
 a2 = DiGraph(adjmx2)
 
 tests = [
+    "randgraphs",
```

Comment:
Why are we moving randgraphs up in the test suite? (Was this primarily for convenience?)